src/Symfony/Component/Serializer/Normalizer/AbstractNormalizer.php

```diff
@@ -265,6 +264,19 @@ protected function getAllowedAttributes($classOrObject, array $context, bool $at
         return $allowedAttributes;
     }
 
+    /**
+     * @return array|false False on no group specified
```

Comment:
False when no group is specified

Reply:
btw, shouldn't we use `null`?

Reply:
We need to double-check this, as `false` was explicitly used in [`getAllowedAttributes`](https://github.com/symfony/symfony/blob/996e5101e15ffaeb38eec0e3df4a464ed494b6cc/src/Symfony/Component/Serializer/Normalizer/AbstractNormalizer.php#L252), there is probably a reason.

Reply:
I don't know about `getAllowedAttributes`, but this now an API. Using `null` would allow adding a return type.